<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayersWdgt</class>
 <widget class="QWidget" name="PlayersWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>250</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Player</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>256</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>PROPERTIES</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="1" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinBoxLAYER">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelLAYERSIZE">
        <property name="text">
         <string>LAYER SIZE</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSpinBox" name="spinBoxLAYERSIZE">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelNAME">
        <property name="text">
         <string>NAME</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelLAYER">
        <property name="text">
         <string>HIDDEN LAYER</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QPushButton" name="pushButtonCOLOR">
        <property name="text">
         <string>COLOR</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEditNAME">
        <property name="text">
         <string>Player1</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QPushButton" name="pushButtonSAVE">
        <property name="text">
         <string>&amp;save</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="pushButtonLOAD">
        <property name="text">
         <string>&amp;load</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButtonCLOSE">
     <property name="text">
      <string>&amp;close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="NetWdgt" name="widgetNET" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NetWdgt</class>
   <extends>QWidget</extends>
   <header>netwdgt.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCLOSE</sender>
   <signal>released()</signal>
   <receiver>PlayersWdgt</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
